test/dummy_webpacker3/app/views/pages/show.html.erb

```diff
@@ -4,7 +4,7 @@
 </ul>
 
 <div id='component-parent'>
-  <%= react_component 'GreetingMessage', { name: @name, last_name: "Last #{@name}", info: { name: @name } }, { id: 'component', class: "greeting-message", prerender: @prerender } %>
+  <%= react_component 'GreetingMessage', { name: @name, lastName: "Last #{@name}", info: { name: @name, lastName: "Last #{@name}" } }, { id: 'component', class: "greeting-message", prerender: @prerender } %>
```

Comment:
Can you do the same in the other dummy applications?
That should make the tests pass.
```
dummy_sprockets
dummy_webpacker1
dummy_webpacker_2
```

Reply:
Can I ask you to also get the one in this file please?
```
https://github.com/reactjs/react-rails/blob/master/test/dummy_sprockets/app/views/pages/show.html.erb
```